csvimport.json

```diff
@@ -0,0 +1,282 @@
+{
+    "appid": "1b0e5665-faeb-4c8e-9fd3-8627cd90d530",
+    "name": "CSV Import",
+    "description": "Ingest CSV files into Phantom",
+    "type": "information",
+    "product_vendor": "Splunk",
+    "logo": "logo_splunk.svg",
+    "logo_dark": "logo_splunk_dark.svg",
+    "product_name": "CSV Import",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
+    "license": "Copyright (c) 2022 Splunk Inc.",
+    "app_version": "1.0.0",
+    "utctime_updated": "2018-11-13T22:04:02.279194Z",
```

Comment:
Seems inaccurate

Reply:
@alexa-phantom yes UTC time will automatically gets updated when it got merged into the next branch.